arch/arm64/src/imx9/imx9_usbdev.c

```diff
@@ -776,8 +780,9 @@ static inline void imx9_writedtd(struct imx9_dtd_s *dtd,
 
   up_clean_dcache((uintptr_t)dtd,
                   (uintptr_t)dtd + sizeof(struct imx9_dtd_s));
-  up_clean_dcache((uintptr_t)data,
-                  (uintptr_t)data + nbytes);
+
```

Comment:
I believe the original line is correct, except for CACHE_ALIGN_UP.

The *3 should not be needed

Perhaps the whole buffer should be invalidated (or cleaned) when it is allocated in imx9_epallocbuffer.